docs/whats-new/staging-release-notes.md

```diff
@@ -2,6 +2,35 @@
 
 > **Tip**: Click the triangle to view more details about the change.
 
+## Staging release 29 March 2023
+
+Frontend version: 1.0.0-20230322.1044 | Backend version: 1.55.6-230323.1031
```

Comment:
the backend version is 1.55.7-230328.0536

Reply:
updated in new commit @JLeeChan